docs/tutorials/auth.md

```diff
@@ -133,9 +133,91 @@ following picture:
 
 ## OpenID Connect
 
-### UI authcode flow
+OpenID Connect is an OAuth2.0 based authentication and authorization mechanism supported by
+several providers. Skipper can act as a proxy for backend server which requires authenticated clients. Skipper handles the authentication with the provider and upon sucessful completion of authentication passes subsequent requests to the backend server.
+
+Skipper's implementation of OpenID Connect Client works as follows:
+
+1. Filter is initialized with the following parameters:
+    1. Secrets file used for encrypting the token in a cookie and also for generating shared secret.
```

Comment:
I think you should show a reason why a shared secret is required here.

Reply:
I've explained the use of the secret at the very end.